libs/pmp-web/auth/login-feature/src/lib/containers/login-page/login-page.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { AvailableSystems } from '@pimp-my-pr/pmp-web/shared/domain';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'pimp-my-pr-login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss']
+})
+export class LoginPageComponent implements OnInit {
+  constructor(private router: Router) {}
+
+  systems = [
+    {
+      imageSrc: 'assets/images/github_logo.png',
+      name: AvailableSystems.github,
+      color: '#2cbe4e'
+    },
+    {
+      imageSrc: 'assets/images/bitbucket_logo.png',
+      name: AvailableSystems.bitbucket,
+      color: '#0052CC'
+    }
+  ];
+
+  ngOnInit(): void {}
+
+  login(system: string): void {
+    switch (system) {
+      case AvailableSystems.github:
+        this.router.navigate(['user']);
+        break;
+      case AvailableSystems.bitbucket:
+        this.router.navigate(['user']);
```

Comment:
case AvailableSystems.bitbucket:
case AvailableSystems.bitbucket:
        this.router.navigate(['user']);
        break;